pkg/core/entities/validator_state_test.go

```diff
@@ -62,11 +62,3 @@ func TestRegisteredAndHasVotes_NotRegisteredWithVotes(t *testing.T) {
 	}
 	require.False(t, state.RegisteredAndHasVotes())
 }
-
-func getDefaultValidator() *ValidatorState {
```

Comment:
Not sure why this one was deleted in ad808b37b3e99b2b0793dbebe6bbf13f7c5704ef.

Reply:
fixed